borg/testsuite/archiver.py

```diff
@@ -892,6 +892,13 @@ def test_list_prefix(self):
         self.assert_in('test-2', output)
         self.assert_not_in('something-else', output)
 
+    def test_list_list_format(self):
+        self.cmd('init', self.repository_location)
+        self.cmd('create', self.repository_location + '::test', src_dir)
+        output_1 = self.cmd('list', self.repository_location)
+        output_2 = self.cmd('list', '--list-format="{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}"', self.repository_location)
+        self.assertEqual(output_1, output_2)
+
```

Comment:
> I think we are missing some comment's because I replied and provided
> with more and alternative test messages. And with a question should I
> touch this now or leave it as it is.
> I'll add a third list-format cmd that should not equal. This should show
> that same output can be generated and list-format options does something.

I don't even see this comment when I follow the link in the mail.
something is borked on github. :(

Yes, please add a test that uses different format.